app/src/ui/lib/sandboxed-markdown.tsx

```diff
@@ -124,8 +124,23 @@ export class SandboxedMarkdown extends React.PureComponent<
     ) {
       return
     }
-    const docEl = frameRef.contentDocument.documentElement
-    const divHeight = docEl.clientHeight
+
+    /*
+     * We added an additional wrapper div#content around the markdown to
+     * determine a more accurate scroll height as the iframe's document or body
+     * element was not adjusting it's height dynamically when new content was
+     * provided.
+     */
+    const docEl = frameRef.contentDocument.documentElement.querySelector(
+      '#content'
+    )
+    if (docEl === null) {
+      return
+    }
+
+    // Not sure why the content height != body height exactly. But, 50px seems
+    // to prevent scrollbar/content cut off.
+    const divHeight = docEl.clientHeight + 50
```

Comment:
Maybe @niik's deeper knowledge of CSS/HTML can solve this mystery. I'm equally perplexed and reminds me of my times with AutoLayout (_shivers_ 😰 )